yaas/blueprint.py

```diff
@@ -13,7 +13,7 @@ class Blueprint:
     def __init__(self, client):
         self.client = client
 
-    def add(self, blueprint, blueprint_name, validate_topology=True):
+    def add(self, blueprint_name, blueprint, validate_topology=True):
```

Comment:
In light of https://github.com/dmtucker/yaas/issues/21#issuecomment-172371718, I think your change is right.